google-cloud-firestore/src/main/java/com/google/cloud/firestore/FirestoreOptions.java

```diff
@@ -278,6 +305,12 @@ public void refresh() {}
   protected FirestoreOptions(Builder builder) {
     super(FirestoreFactory.class, FirestoreRpcFactory.class, builder, new FirestoreDefaults());
 
+    if (builder.openTelemetryOptions == null) {
```

Comment:
Nit: use ternary operator as L314 and L319 for a consistent style?

Reply:
Good idea! Done.